### PR TITLE
Add new selenium test case for create new paragraph button

### PR DESCRIPTION
**Pull request by ravicodder**
### What is this PR for?
Add a new test case for testing the create new button.
Create new button is for creating new paragraph within notebook

### What type of PR is it?
Test

### Is there a relevant Jira issue?

NA

### How should this be tested?

On OSX, you'll need firefox 42.0 installed, then you can run with

PATH=~/Applications/Firefox.app/Contents/MacOS/:$PATH CI="" \
mvn -Dtest=org.apache.zeppelin.ParagraphActionsIT -Denforcer.skip=true \
test -pl zeppelin-server

### Questions:
* Does the licenses files need update?NO
* Is there breaking changes for older versions?NO
* Does this needs documentation?NO

### CODE REVIEW

File: zeppelin-server/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java [hunk 1]
```diff
@@ -55,6 +56,33 @@ public void tearDown() {
...
+  @Test
+  public void testCreateNewButton() throws InterruptedException {
+    if (!endToEndTestEnabled()) {
+      return;
+    }
+    try {
+      createNewNote();
+
+      waitForParagraph(1, "READY");
+      Integer nosOfParas =    driver.findElements(By.xpath("//div[@ng-controller=\"ParagraphCtrl\"]")).size();
```
Comment (prabhjyotsingh) [inline zeppelin-server/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java]: extra space after "="

File: zeppelin-server/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java [hunk 2]
```diff
@@ -55,6 +56,33 @@ public void tearDown() {
...
+      driver.findElement(By.xpath(getParagraphXPath(1) + "//span[@class='icon-settings']")).click();
+      driver.findElement(By.xpath(getParagraphXPath(1) + "//ul/li/a[@ng-click='insertNew()']")).click();
+      waitForParagraph(2, "READY");
+      Integer newNosOfParas =  driver.findElements(By.xpath("//div[@ng-controller=\"ParagraphCtrl\"]")).size();
+      collector.checkThat("The number of new paragraph and old paragraph",
+              (nosOfParas+1==newNosOfParas),
+              CoreMatchers.equalTo(true));
+
```
Comment (prabhjyotsingh) [inline zeppelin-server/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java]: collector.checkThat("The number of new paragraph and old paragraph",
              nosOfParas + 1,
              CoreMatchers.equalTo(newNosOfParas));

**Comment by Leemoonsoo** [comment]
Could this test also cover new paragraph by clicking button between paragraphs?

**Comment by ravicodder** [comment]
@Leemoonsoo This test does not cover new paragraph by clicking + buttons between paragraphs

**Comment by ravicodder** [comment]
@Leemoonsoo Modified test case to cover new paragraph by clicking + buttons between paragraph.

**Comment by Leemoonsoo** [comment]
@ravicodder Thanks for taking care. 

LGTM

**Comment by Leemoonsoo** [comment]
Merge if there're no more discussions

**Comment by prabhjyotsingh** [comment]
@ravicodder I think you should test both append paragraph to top, and append paragraph to bottom buttons as well. Instead of only testing append to top.

https://github.com/apache/incubator-zeppelin/pull/688/files#diff-48409cf8ca5ddb1aea1a62ada3cd091dR82

**Comment by ravicodder** [comment]
@prabhjyotsingh Modified test case to include test case to test append paragraph to top and bottom. 
 Please have a look.

**Comment by prabhjyotsingh** [comment]
CI green, LGTM.

**Comment by Leemoonsoo** [comment]
LGTM and merge if there're no more discussion

**Pull request closed without merging**